src/app/client/src/app/modules/learn/components/course-consumption/course-consumption-header/course-consumption-header.component.ts

```diff
@@ -153,4 +154,9 @@ export class CourseConsumptionHeaderComponent implements OnInit, AfterViewInit,
     this.unsubscribe.next();
     this.unsubscribe.complete();
   }
+  checkExpiredBatch() {
```

Comment:
getBatchStatus () {
return (this.enrolledBatchInfo.status === 2 && this.progress < 100);
}